src/Microsoft.Azure.SignalR.Common/Utilities/JwtTokenCredentials.cs

```diff
@@ -0,0 +1,46 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Rest;
+
+namespace Microsoft.Azure.SignalR.Management
+{
+    class JwtTokenCredentials : ServiceClientCredentials
```

Comment:
add protection level

Reply:
fixed in [fix](https://github.com/Azure/azure-signalr/pull/992/commits/d58ec08f9e255560335ffc4bdf60be844ca8ee9b)